specification/UAVCAN_Specification.tex

```diff
@@ -35,7 +38,7 @@ \section*{Overview}
     \item Efficient exchange of large data structures with automatic decomposition and reassembly.
     \item Lightweight, deterministic, easy to implement, and easy to validate.
     \item Suitable for deeply embedded, resource constrained, hard real-time systems.
-    \item Doubly- or triply- redundant CAN bus support.
+    \item Supports doubly- and triply-redundant transports.
```

Comment:
Is this wording ambigous? I think double redundant may mean 3 can busses (two may fail) and triply redundant may mean 4 (three may fail)?

Reply:
Rephrased